packages/textmate-grammars/data/go.tmLanguage.json

```diff
@@ -0,0 +1,676 @@
+{
```

Comment:
I think this file is copied from somewhere? If so a CQ is required. See: https://github.com/theia-ide/theia/wiki/Registering-CQs#case-3rd-party-project-code-copiedforked-from-another-project-into-eclipse-theia-maintained-by-us

e.g.: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=18337

Reply:
This one is  copied from
https://github.com/Microsoft/vscode/blob/master/extensions/go/syntaxes/go.tmLanguage.json
I will create CQ

Reply:
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=18775

Reply:
@tolusha You can use the same CQ for both this and what was copied from vscode's 'language-configuration.json', since they're from the same upstream project. You can just update the attached archive. For that later file, there is license info in same directory in file `cgmanifest.json`, so I would attach it also.

Reply:
@marcdumais-work 
Thank you. I've updated CQ.